pyshacl/validate.py

```diff
@@ -406,6 +429,7 @@ def validate(
     allow_warnings: Optional[bool] = False,
     max_validation_depth: Optional[int] = None,
     sparql_mode: Optional[bool] = False,
+    focus_nodes: Optional[List[Union[str | URIRef]]] = None,
```

Comment:
`Union[str | URIRef]` raises a syntax error pre-Python-3.10.  How about `Union[str, URIRef]`?

Reply:
Thank you!
I blame that on my new LLM-based code-completion assistant, it must've added the typing in that manner, I don't remember doing so.